clang/Type.d

```diff
@@ -4,73 +4,90 @@
  * Version: Initial created: Jan 29, 2012
  * License: $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost Software License 1.0)
  */
+
```

Comment:
I prefer that there's no newline between the documentation and the symbol it's attached to.